sdk_contrib/hapi/lib/plugin.js

```diff
@@ -0,0 +1,33 @@
+const xray = require('./xray');
+
+/**
```

Comment:
I think this doc should be on the method rather than the module. But either way, it looks like we can leave it out if it's just defining the plugin's interface provided by Hapi but not anything a user would call.

Reply:
@anuraaga Good call, I just went ahead and pulled it.